adb-utils.spec

```diff
@@ -1,6 +1,6 @@
 Name:          adb-utils
 Version:       1.5
-Release:       1%{?dist}
+Release:       2%{?dist}
```

Comment:
If agreed on above comment (https://github.com/projectatomic/adb-utils/pull/108/files#r60394557), the release bump is not required.